doc/explanation/tour/decoding.rst

```diff
@@ -0,0 +1,68 @@
+Parsing of Dune Files
+---------------------
+
+Parsing ``dune`` files is done in two steps: first, they are parsed as
```

Comment:
There’s nothing incorrect about the sentence structure here, but I think it would be more readable/scannable if it was in a list: 

Parsing `dune` files is done in two steps:
1. They are parsed…
2. Then they are decoded…

But it’s totally up to you! Just my two cents 🙂